scripts/contentScript.js

```diff
@@ -0,0 +1,19 @@
+let starredUsers = [];
+
+chrome.storage.sync.get('starredUsers', function(data) {
```

Comment:
Use arrow function syntax instead- it is more modernly used

() => { statements }